charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml

```
@@ -16,9 +17,10 @@ items:
       {{ $.Values.grafana.sidecar.dashboards.label }}: {{ ternary $.Values.grafana.sidecar.dashboards.labelValue "1" (not (empty $.Values.grafana.sidecar.dashboards.labelValue)) | quote }}
       {{- end }}
       app: {{ template "kube-prometheus-stack.name" $ }}-grafana
-{{ include "kube-prometheus-stack.labels" $ | indent 6 }}
+      {{ include "kube-prometheus-stack.labels" $ | indent 6 }}
```

This needs an nindent and not an indent I think
```suggestion
      {{ include "kube-prometheus-stack.labels" $ | indent 6 }}
```